test/common/http/utility_fuzz_test.cc

```diff
@@ -79,7 +79,25 @@ DEFINE_PROTO_FUZZER(const test::common::http::UtilityTestCase& input) {
   }
   case test::common::http::UtilityTestCase::kInitializeAndValidate: {
     const auto& options = input.initialize_and_validate();
-    Http2::Utility::initializeAndValidateOptions(options);
+    try {
+      Http2::Utility::initializeAndValidateOptions(options);
+    } catch (EnvoyException& e) {
+      std::string msg = e.what();
```

Comment:
nit: you can also use an `absl::string_view` to avoid the string allocation here, since all it's needed for is comparisons which can be done using `absl::StartsWith` as Asra suggested.

Reply:
good idea, thanks! done